includes/wikia/parser/templatetypes/handlers/TemplateCleanup.class.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class TemplateCleanup {
```

Comment:
I'm not sure if the name covers what this class is responsible for - maybe DocumentCleanup or HtmlCleanup would be more informative